deploy/profiles/legion-ci.epm.kharlamov.biz.yml

```diff
@@ -75,7 +75,7 @@ storageclass: efs
 airflow_dags_dir: '/airflow-dags'
 airflow_dags_pvc: legion-airflow-dags
 airflow_s3_bucket_name: 'epm-legion-data-ci'
-airflow_s3_log_url: "s3://{{ airflow_s3_bucket_name }}/{{ enclave }}/airflow-logs/"
+airflow_s3_log_url: "s3://airflow-logs/"
```

Comment:
Looks like all enclaves at all clusters will log to the same location? Shouldn't there be  a separate log storage per cluster/enclave?

Reply:
Airflow will write log files to same directory:
*s3://{{ airflow_s3_bucket_name }}/{{ enclave }}/airflow-logs/*
But it's configured in different way:
1. In airflow configuration: *s3_bucket_path: "{{ enclave }}/"*
2. Bucket name is specified in Airflow connection in secret file.

Reply:
worth renaming variable so